src/Inkluzitron/Modules/BdsmTestOrg/GraphPainter.cs

```diff
@@ -0,0 +1,272 @@
+﻿using DC = Discord;
+using Inkluzitron.Data.Entities;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Globalization;
+
+namespace Inkluzitron.Modules.BdsmTestOrg
+{
+    public class GraphPainter
```

Comment:
Udělal bych z toho normálně službu, která bude v kontejneru. + implementoval bych IDisposable. Hrozí, že to zde bude těžce leakovat (fonty, brushe, ...)

Reply:
Udělána z toho služba co je registrovaná v DI.